.travis.yml

```diff
@@ -57,8 +58,6 @@ deploy:
   - provider: script
     script: "./build/push-javadoc-to-gh-pages.sh"
     skip_cleanup: true
-    on:
```

Comment:
Hmmm, I don't think we want to deploy on all builds, do we?   That's why I had set the `on: tags` qualifier.

Reply:
The script was designed to deploy branches too
See: http://watson-developer-cloud.github.io/java-sdk/